databuilder/tests/unit/models/test_application.py

```diff
@@ -26,7 +28,7 @@
 
 @dataclass
 class ApplicationTestCase:
```

Comment:
Notice that the only change here is migrating non-airflow apps to `GenericApplication`. Everything else is fully bw-compatible. 

Since extension of this model to non-airflow is recent (mid-August), disruption is likely minimal.